access-om3/ww3/mod_def/global.1deg/2024.04.18/manifest.yaml

```diff
@@ -0,0 +1,13 @@
+format: yamanifest
+version: 1.0
+---
+manifest.yaml:
+  fullpath: /g/data/vk83/configurations/inputs/access-om3/ww3/mod_def/global.1deg/2024.04.18/manifest.yaml
+  hashes:
+    binhash: 506fe562f9b3f966d163261c3359a581
+    md5: bb031c32181b5fc574c9f7141bafd6c7
```

Comment:
Seem to have added an entry for the manifest itself.

Reply:
Fixed via https://github.com/ACCESS-NRI/model-config-inputs/pull/6/commits/efad4afd534eb76f9fe3331bbdb62d11646babd7